src/Umbraco.Web/PropertyEditors/SliderPropertyEditor.cs

```diff
@@ -32,9 +35,38 @@ internal class SliderPreValueEditor : PreValueEditor
             [PreValueField("step", "Step increments", "number")]
             public int StepIncrements { get; set; }
 
-            [PreValueField("orientation", "Orientation", "views/propertyeditors/slider/orientation.prevalues.html")]
-            public string Orientation { get; set; }
+            [PreValueField("precision", "Precision", "number", Description = "The number of digits shown after the decimal. Defaults to the number of digits after the decimal of step value.")]
+            public int Precision { get; set; }
+
+            [PreValueField("handle", "Handle", "views/propertyeditors/slider/handle.prevalues.html", Description = "Handle shape. Default is 'round\'")]
+            public string Handle { get; set; }
+
+            [PreValueField("tooltip", "Tooltip", "views/propertyeditors/slider/tooltip.prevalues.html", Description = "Whether to show the tooltip on drag, hide the tooltip, or always show the tooltip. Accepts: 'show', 'hide', or 'always'")]
+            public string Tooltip { get; set; }
+
+            [PreValueField("tooltipSplit", "Tooltip split", "boolean", Description = "If false show one tootip if true show two tooltips one for each handler")]
+            public string TooltipSplit { get; set; }

```

Comment:
Ideally this would be a bool. I am not sure why ```$scope.model.config.tooltipSplit``` then is undefined in AngularJS controller, when using a bool for PreValueField?